nativeunwind/stackdeltatypes/stackdeltatypes.go

```diff
@@ -29,10 +29,11 @@ const (
 	UnwindOpcodeFlagDeref uint8 = C.UNWIND_OPCODEF_DEREF
 
 	// UnwindCommands from the C header file
-	UnwindCommandInvalid int32 = C.UNWIND_COMMAND_INVALID
-	UnwindCommandStop    int32 = C.UNWIND_COMMAND_STOP
-	UnwindCommandPLT     int32 = C.UNWIND_COMMAND_PLT
-	UnwindCommandSignal  int32 = C.UNWIND_COMMAND_SIGNAL
+	UnwindCommandInvalid      int32 = C.UNWIND_COMMAND_INVALID
```

Comment:
Maybe we should bring these constants into `support/types_def.go` - then we don't need to manually keep them in sync. But this can be part of another PR.

Reply:
Sure, will do a follow up PR on this.